conf/nginx.conf

```diff
@@ -1,28 +1,28 @@
-location / {
+#sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
+location __PATH__ {
 
-  # Path to source
-  alias __FINALPATH__/;
-
-  index index.php;
-
-  try_files $uri /public/$uri /index.php;
+  root /var/www/;
```

Comment:
Hmmm pretty sure the subpath is related to this ? Why use `root /var/www/` instead of the usual `alias __FINALPATH__/` ?

Reply:
Hi @alexAubi, thank for your help ❤️ 
In fact I'm really not a NGINX expert. So I simply copy/paste this "root" from [Selfoss documentation](https://github.com/fossar/selfoss/wiki/nginx-configuration)...